packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -149,9 +144,10 @@ Router = Router.extend({
         CompleteResetPasswordView,
         {
           ...Url.searchParams(this.window.location.search, [
+            'code',
             'email',
             'emailToHashWith',
-            'code',
+            'service',
```

Comment:
Kind of wonder if we should just send everything through similar to other views we've converted, but I think this covers it all. Interesting that Sync PW reset password works as expected without this...? But then again we're ending all the reset PW flows with the same end-screen so maybe that's why.

Reply:
Hm, it looks like this array isn't actually working as expected anyway. `service` still gets passed along even if it's not included in the array of picked params... I didn't dig into why, but might as well just send everything as we do for other views.

Reply:
Oh, I know why. It's because we're copy pasting this link and not coming in from another content-server page. It's not actually hitting `router.js`...

Reply:
I think prior to this PR it would still hit `router.js` but it doesn't now since we check at the server-level if it should be enabled? So seems fine to leave this or remove the array 👍